examples/parser/package.json

```diff
@@ -2,7 +2,11 @@
     "name": "chevrotain_examples_parser",
     "version": "0.1.0",
     "scripts": {
-        "test": "grunt --gruntfile minification/gruntfile.js && cd webpack && yarn install && yarn bundle && rm -rf node_modules && cd .. && mocha \"!(node_modules)/**/*spec.js\""
+        "build_minification": "grunt --gruntfile minification/gruntfile.js",
+        "build_webpack": "cd webpack && yarn install && yarn bundle && rm -rf node_modules && cd ..",
+        "build_serialized_grammar": "node ./serialized_grammar/gen_serialized.js",
+        "pretest": "npm run build_minification && npm run build_serialized_grammar && npm run build_webpack",
+        "test": "NODE_ENV=production mocha \"!(node_modules)/**/*spec.js\""
```

Comment:
maybe we should set the env variable (production) be defined in the tests so the example would be more self contained.